### PR TITLE
fix: add missing config file

### PR DESCRIPTION
**Pull request by QuakeEye**

### CODE REVIEW

File: release.config.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        
+        
+        // Theoretical scenario for possible extension in which the project
+        //  has its version set and is published respectively with the following
+        //  commands:
+        
+        // ,
+        // {
+        //   "prepareCmd": "set-version ${nextRelease.version}",
+          // "publishCmd": "publish-package"
```
Comment (LuukBerkers) [inline release.config.js:17]: Deze comment is niet aligned

**Review by JoenvandeVorle — DISMISSED** [review]

**Review by LuukBerkers — CHANGES_REQUESTED** [review]

**Comment by QuakeEye** [comment]
@LuukBerkers @JoenvandeVorle @joachimdekker What is this weird language you are all speaking?

**Review by joachimdekker — APPROVED** [review]
LGTM!

**Review by LuukBerkers — APPROVED** [review]
lgtm

**Review by JoenvandeVorle — APPROVED** [review]
sfime (seems fine in my eyes)

**Comment by joachimdekker** [comment]
Kan je die message aanpassen?

**Pull request merged**